src/main/java/com/google/cloud/resourcemanager/Lien.java

```diff
@@ -181,11 +179,6 @@ public final int hashCode() {
     return Objects.hash(super.hashCode(), options);
   }
 
-  private void readObject(ObjectInputStream in) throws IOException, ClassNotFoundException {
```

Comment:
This looks private but serialization is weird and it might actually matter to someone.

Reply:
done